src/components/dialogs/criteria-based-filter-dialog-content.js

```diff
@@ -83,12 +84,80 @@ export const FilterTypeSelection = ({
     );
 };
 
+const backToFrontTweak = (response) => {
```

Comment:
maybe you could comment these 2 functions. They are very "technical", it's hard to follow what they really do

Reply:
It just adapts the data structure from what is received from the backend to the form that the component can handle.